scripts/genie.lua

```diff
@@ -563,6 +562,11 @@ or _OPTIONS["with-combined-examples"] then
 		, "41-tess"
 		)
 
+	-- 17-drawstress requires multithreading, does not compile for singlethreaded wasm
+--	if platform is not single-threaded then
+		exampleProject(false, "17-drawstress")
+--	end
+
```

Comment:
This is a todo before landing: was not able to figure out, how can I disable this example on Emscripten singlethreaded builds? Other examples seem to build ok.

Reply:
I can modify example to not require threading... I'll merge as is, and then fix it.